### ISSUE TITLE
[Bug] [Mistwood] [Mayberry Matriarch] Priority over a Wife

### ISSUE THREAD

**Issue by Dajakk** [tadeaustria/EverdellScore#45]
Replication scenario (possible in real game):
- build a Husband
- build a Wife (pair it with a Husband)
- build a second Wife, this one is not paired
- use second Wife to build Mayberry Matriarch, it should not be paired with a Husband

Right now in the app, the first Wife that was paired stops giving extra points, but Mayberry Matriarch does not give extra points, because there is no Farm. Legendary always takes precedence over normal wife. Ideally it's the player who decides which wife is paired.
If the player knows he is not able to play a farm into his town, he might decide it's more profitable for him, to use a Wife that is not paired for a Legendary card.

The easiest way of fixing this issue would be introducing one more card for a wife - "wife with a husband"
Another one would be introducing some sort of a checkbox or a slider on a Wife card to determine which wives are paired.

**Comment by Dajakk** [comment]
What's interesting, in Polish version of the rule book there is a part that's missing from English one, but it was later clarified on BGG by a Designer. It reads: "You should decide on marriage when playing a card, and that decision cannot be changed later on."

![image](https://github.com/tadeaustria/EverdellScore/assets/471436/2437e16f-ece2-45f4-a49a-78cd5f31afcc)

**Comment by Dajakk** [comment]
One more possible solution:
There could be a new card called "Husband & Wife", but it would need to have 2 types to work properly with for example visitors, both Production and Prosperity

Some of those solutions have additional advantages, like simplifying calculations or possibility to get rid of wife prosperity section, this could get moved to a card itself

**Comment by tadeaustria** [comment]
Yes, I am aware of this bug. It kinda developed through the fact, that for the base game it did not matter.
But yes, this whole calcuation for the wives are a bit messy.

The slider sultion is currently no option, as there is no place to store such information. Currently cards in town are just references, not copies.

**Comment by tadeaustria** [comment]
The more I think about, the more I like the idea of the separate card "wife with a husband". It would make stuff alot easier, but a little bit more complicated for the end user.

**Comment by Dajakk** [comment]
Personally I think "wife with a husband" is easiest to implement and simplifies everything the most. Of course it would be responsibility of a player to make sure, that this card is always paired with a husband.
On the other hand "wife + husband" would be the safer option for a player, but on our side it would require some changes:
- this card should be counted as 2 for some effects
- it should be considered both production and governance for visitors and other governance cards

Anyways, for both approaches similar thing should probably be implemented for Mayberry card.

**Comment by tadeaustria** [comment]
No, I don't like _wife + husband_, it is not necessary and should not be necessary to support multiple types on one card.
